From 3c156e258203f833ffbc6cd5a012cf58632dc70c Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 28 Feb 2025 11:49:57 +0100
Subject: [PATCH] arm64: dts: st: remove label for led-blue in stm32mp257f-ev1
 board

Remove the deprecated property 'label' for led-blue description in
stm32mp257f-ev1 board device tree.

Fixes: 1219058fc44b ("arm64: dts: st: add heartbeat LED node in stm32mp257f-ev1")
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I67a92d2b5cc5bf05492929baf9272ed566bcffbc
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/440776
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -92,7 +92,6 @@
 		compatible = "gpio-leds";
 
 		led-blue {
-			label = "heartbeat";
 			function = LED_FUNCTION_HEARTBEAT;
 			color = <LED_COLOR_ID_BLUE>;
 			gpios = <&gpioj 7 GPIO_ACTIVE_HIGH>;
